### PR TITLE
make search clearable after selecting 

### PR DESCRIPTION
**Pull request by yCobanoglu**
make async-select clearable. Clearing the async-select passes a `null` to the handleChange function.

### CODE REVIEW

**Comment by yCobanoglu** [comment]
so in Fragmentarium search is clearable which is better. But in ReferenceEditor it is not because there we can just delete it instead of Clearing it.

File: src/bibliography/ui/BibliographySelect.tsx [hunk 1]
```diff
@@ -45,13 +45,14 @@ export default class BibliographySelect extends Component<
     this.setState({
       selectedOption,
     })
-    this.props.onChange(selectedOption.entry)
+    this.props.onChange(selectedOption)
```
Comment (jlaasonen) [inline src/bibliography/ui/BibliographySelect.tsx:48]: Why do you leak implementation details to the event handler? The argument of `onChange` should be the new value (i.e. same type as the `value`), not any data structure specific to the current implementation.

**Review by jlaasonen — CHANGES_REQUESTED** [review]

File: src/bibliography/ui/BibliographySelect.tsx (hunk 1 above)
Comment (yCobanoglu) [reply src/bibliography/ui/BibliographySelect.tsx:48]: i created a new PR dealing with this. [https://github.com/ElectronicBabylonianLiterature/ebl-frontend/pull/41](url)

**Comment by yCobanoglu** [comment]
New PR for this:
https://github.com/ElectronicBabylonianLiterature/ebl-frontend/pull/41

**Pull request closed without merging**